package.json

```diff
@@ -8,6 +8,7 @@
     "start-remote": "node scripts/check-local-config && CONFIG_FILES=server/config/local.json,server/config/content.json grunt server",
     "postinstall": "bower update --config.interactive=false -s",
     "test": "intern-runner config=tests/intern",
+    "test-circle": "intern-runner config=tests/intern_functional_circle fxaAuthRoot=https://fxa-ci.dev.lcip.org/auth fxaEmailRoot=http://restmail.net fxaOauthApp=https://oauth-fxa-ci.dev.lcip.org fxaUntrustedOauthApp=https://321done-fxa-ci.dev.lcip.org fxaProduction=true",
```

Comment:
I moved all the extra config from circle.yml here so that we could start the local server using the `tests/ci/circle_config.json` and run the tests like Circle would by typing `npm run test-circle`.